shaders/src/light_indirect.fs

```diff
@@ -676,14 +671,26 @@ void applyRefraction(
     float lod = max(0.0, 2.0 * log2(tweakedPerceptualRoughness) + frameUniforms.refractionLodOffset);
 
     vec4 Fat = textureLod(light_ssr, p.xy, lod);
-    float at = Fat.a;
-#if defined(IS_TRANSPARENT_OR_REFRACTIVE)
-    vec3 Ft = mix(vec3(1.0), Fat.rgb, at);
-#else
-    vec3 Ft = Fat.rgb;
```

Comment:
nit: You could've kept an `vec3 Ft = Fat.rgb; float at = Fat.a;` assignment here and left `Ft`, `at` untouched above. Then you wouldn't need an else branch to `frameUniforms.needsAlphaChannel == 1.0` either.